src/App.js

```diff
@@ -1,14 +1,70 @@
-import React from 'react';
+import React, {useState} from 'react';
 import {Navigate, Route, Routes} from "react-router-dom";
 import useAuth from "auth/useAuth";
 import {CssBaseline, ThemeProvider} from "@mui/material";
 import {defaultTheme} from "assets/styles";
-import Home from "pages/Home";
 import Login from "pages/Login";
 import SignUp from "pages/SignUp";
 import ResetPassword from "pages/ResetPassword";
 import Footer from "components/Footer";
 import Error from "pages/Error";
+import Home from "pages/Home/Home";
+import highlight_1 from '../src/assets/images/highlight_1.svg';
```

Comment:
Most icons have meaning, I'd find it easier to identify these icons if they were named for their purpose. For example `highlight_1.svg` could be named `highlight_effective_knowledge_testing.svg`

Reply:
Done &#9989;